### PR TITLE
helprank/FE: /helprank

### PR DESCRIPTION
**Pull request by raymondng**
Another baby step

I model this after how we currently handle tag-keeps -- it seems reasonably straightforward. Would appreciate if you can take a look and let me know if I have missed anything. 

@andrewconner @stkem @martinraison

### CODE REVIEW

**Comment by stkem** [comment]
lgtm overall, however a look by @andrewconner or @martinraison would be good, as I have not set up a new route before.

**Comment by martinraison** [comment]
looks good!

**Comment by raymondng** [comment]
Thanks! It's still very rough but I'll merge and iterate.

File: kifi-backend/modules/shoebox/angular-black/src/helprank/helpRank.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    $scope.scrollDistance = '100%';
+    $scope.scrollDisabled = false;
+
+    $scope.getNextKeeps = function () {
+      if ($scope.loading) {
+        return;
+      }
+
+      $scope.loading = true;
+      return keepService.getKeepsByHelpRank(helprank).then(function (list) {
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular-black/src/helprank/helpRank.js:63]: Needs pagination. Takes 5s to show this page for me (3s download, 2s render). ie, smaller pages (not 40 keeps at once)

Comment (raymondng) [inline kifi-backend/modules/shoebox/angular-black/src/helprank/helpRank.js:63]: Yup already fixed this locally.

**Pull request merged**